redbot/core/utils/common_filters.py

```diff
@@ -16,7 +16,7 @@
 # regexes
 URL_RE = re.compile(r"(https?|s?ftp)://(\S+)", re.I)
 
-INVITE_URL_RE = re.compile(r"(discord\.(?:gg|io|me|li)|discordapp\.com\/invite)\/(\S+)", re.I)
+INVITE_URL_RE = re.compile(r"(discord\.(?:gg|io|me|li|com)|discordapp\.com\/invite)\/(\S+)", re.I)
```

Comment:
The invites at discord.com need the `/invite` part same as discordapp.com
You should instead make the `app` part of the discordapp.com optional

Reply:
This edit has been added